# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1327431986 -3600
#      Tue Jan 24 20:06:26 2012 +0100
# Branch gui
# Node ID 6e6194a7b102f32be474fedb49ecef6c5ab6037b
# Parent  605675153f1d08029a8248ce147670da49b5252f
Added subscripts for retrieeving qterminal and qirc int compile.sh. Added copyright headers.

diff --git a/gui/compile.sh b/gui/compile.sh
--- a/gui/compile.sh
+++ b/gui/compile.sh
@@ -1,18 +1,23 @@
-# Quint - A graphical user interface for Octave
-# Copyright (C) 2011 Jacob Dawid
-# jacob.dawid@googlemail.com
+#!/bin/bash
+
+# OctaveGUI - A graphical user interface for Octave
+# Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-qmake-qt4 && make
+./get-qterminal.sh
+./get-qirc.sh
+qmake-qt4 octave-gui.pro
+make
diff --git a/gui/get-qirc.sh b/gui/get-qirc.sh
--- a/gui/get-qirc.sh
+++ b/gui/get-qirc.sh
@@ -1,10 +1,27 @@
 #!/bin/bash
 
+# OctaveGUI - A graphical user interface for Octave
+# Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
 qirc_dir="qirc";
 echo "Updating qirc..";
 
 # Check whether we have already cloned the repository:
 if [ -d $qirc_dir ]; then
 	# Yes, so just pull any changes and rebuild.
 	cd qirc
 	git pull
diff --git a/gui/get-qterminal.sh b/gui/get-qterminal.sh
--- a/gui/get-qterminal.sh
+++ b/gui/get-qterminal.sh
@@ -1,10 +1,27 @@
 #!/bin/bash
 
+# OctaveGUI - A graphical user interface for Octave
+# Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
 qterminal_dir="qterminal";
 echo "Updating qterminal..";
 
 # Check whether we have already cloned the repository:
 if [ -d $qterminal_dir ]; then
 	# Yes, so just pull any changes and rebuild.
 	cd qterminal
 	git pull
diff --git a/gui/start.sh b/gui/start.sh
--- a/gui/start.sh
+++ b/gui/start.sh
@@ -1,20 +1,22 @@
-# Quint - A graphical user interface for Octave
-# Copyright (C) 2011 Jacob Dawid
-# jacob.dawid@googlemail.com
+#!/bin/bash
+
+# OctaveGUI - A graphical user interface for Octave
+# Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
+# GNU Affero General Public License for more details.
 #
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
 
 LANG=en_US.UTF-8
 cd bin
 ./octave-gui
